### ISSUE TITLE
OPLA-Lahore Allama Iqbal International Airport-PUNJAB-PAKISTAN

### ISSUE THREAD

**Issue by luqmanhussain** [InfiniteFlightAirportEditing/Airports#2460]
Missing everything pretty much except runways. 
Ill fix it.

**Comment by ErshovlaD** [comment]
Can I take this? My previous airports have already been sent to the email

**Comment by brunocr98** [comment]
Sure.

**Comment by moritzbabl** [comment]
Already got the file per mail, isn’t assigned but I don’t see a point in doing it twice

**Comment by luqmanhussain** [comment]
ive already done it. just waiting on approval

Regards,


Luqman Hussain
07508 858049

On 22 Oct 2017, at 18:07, -ErshovlaD- <notifications@github.com<mailto:notifications@github.com>> wrote:


Can I take this? My previous airports have already been sent to the email

**Comment by luqmanhussain** [comment]
[OPLA.zip](https://github.com/InfiniteFlightAirportEditing/Airports/files/1443793/OPLA.zip)
i dont have time to finish this due to other projects. anyone who wants to carry on please find attached apt file, kml file and airport google earth satellite photo.

**Comment by luqmanhussain** [comment]
Latest review items from Moritz:

-airport layout inaccurate:
      -the new taxiway 1 in the east doesn't exist in real life 
      -taxiway parts in the west are missing
-centerline line types are incorrect
-buildings are missing and must be added in the opposite material of the taxiway
-spawn attributes are inaccurate
-spawns are not snapped
-taxiway materials are inaccurate

**Comment by moritzbabl** [comment]
Member inactive, up for grabs

**Comment by Tamz00** [comment]
I’ll grab this for the time being, might do it during the weekend or the upcoming holidays

**Comment by moritzbabl** [comment]
added

**Comment by kilt46** [comment]
![image](https://user-images.githubusercontent.com/14023167/38706447-ca01914e-3e72-11e8-9ac3-3226fcd24aec.png)

**Comment by kilt46** [comment]
Missing gates from OLD TERMINAL
![image](https://user-images.githubusercontent.com/14023167/38707822-a14d8270-3e78-11e8-8d75-d9397d115448.png)
